convert2rhel/utils.py

```diff
@@ -472,10 +472,19 @@ def download_pkg(
     output, ret_code = run_cmd_in_pty(cmd, print_output=False)
     if ret_code != 0:
         loggerinst.warning(
-            "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
+            " you are using an env that overrides an inhibitor and"
+            " you understand that if you continue you might lose the rollback."
+            " \nOutput from the yum downloader call:\n%s" % (pkg, output)
         )
-        return None
+
+        if not CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK in os.environ:  # template for what needs to be done
```

Comment:
Think on this line you forgot to add some `""` in the `CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK`, as it seems you're doing an existence check here. 

Maybe something like this: 

```python
if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:  # template for what needs to be done
```

One thing that crossed my mind right now is: 

* Should we actually validate the value for `CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK` environment variable? 

Because if we do, we could add another condition to this `if-statement` to validate if the value for that variable is the right one.

Reply:
I'm not quite sure which would be better to do in this situation, but I will check with Eric or Michael to see what would be best!

thanks!

Reply:
We don't really need to care about validating the data. Just as long as the env is set that's good enough